src/core/SampleBuffer.cpp

```diff
@@ -296,72 +296,41 @@ void SampleBuffer::update( bool _keep_settings )
 }
 
 
-void SampleBuffer::convertIntToFloat ( int_sample_t * & _ibuf, f_cnt_t _frames, int _channels)
+void SampleBuffer::convertIntToFloat (int_sample_t * & _ibuf, f_cnt_t _frames, int _channels)
 {
 	// following code transforms int-samples into
 	// float-samples and does amplifying & reversing
 	const float fac = 1 / OUTPUT_SAMPLE_MULTIPLIER;
-	m_data = MM_ALLOC( sampleFrame, _frames );
-	const int ch = ( _channels > 1 ) ? 1 : 0;
+	m_data = MM_ALLOC (sampleFrame, _frames);
+	const int ch = (_channels > 1) ? 1 : 0;
 
 	// if reversing is on, we also reverse when
 	// scaling
-	if( m_reversed )
+	int idx = m_reversed ? (_frames - 1) * _channels : 0;
```

Comment:
I think we should do Dom's suggestion here.
```suggestion
	const bool is_reversed = m_reversed;
	int idx = is_reversed ? (_frames - 1) * _channels : 0;
```
and use `is_reversed` for the other, too.

Reply:
Yup, missed this one, thanks.